lib/common/changer.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ReleaseManager
+  module Common
+    class Changer
+      class << self
+        def change_promotion(request)
+          create_component
+          clone_ci_configs
+          git_helper.use_repo(CI_CONFIGS_DIR) do
+            modify_file(request)
+            commit_and_push_changes(request)
+          end
+        end
+
+        private
+
+        def create_component
```

Comment:
This method could be implemented in the factory (eg. factory.create_ci_job_configs)